fastapi/security/api_key.py

```diff
@@ -30,8 +39,17 @@ async def __call__(self, request: Request) -> Optional[str]:
 
 
 class APIKeyHeader(APIKeyBase):
-    def __init__(self, *, name: str, scheme_name: str = None, auto_error: bool = True):
-        self.model: APIKey = APIKey(**{"in": APIKeyIn.header}, name=name)
+    def __init__(
+        self,
+        *,
+        name: str,
+        description: str = None,
+        scheme_name: str = None,
```

Comment:
`        description: Optional[str] = None,
        scheme_name: Optional[str] = None,
	
`

Reply:
@MashhadiNima, thanks for the review. I've applied your improvements.

Reply:
you're welcome. 👨🏻‍💻🍻